cloudify-manager-worker/values.yaml

```diff
@@ -306,3 +332,161 @@ containerSecurityContext:
+    loopback_users.guest = false
+    {{ tpl .Values.extraConfiguration . }}
+    {{- if .Values.auth.tls.enabled }}
+    ssl_options.verify = {{ .Values.auth.tls.sslOptionsVerify }}
+    listeners.ssl.default = {{ .Values.service.tlsPort }}
+    ssl_options.fail_if_no_peer_cert = {{ .Values.auth.tls.failIfNoPeerCert }}
+    ssl_options.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    ssl_options.certfile = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    ssl_options.keyfile = /opt/bitnami/rabbitmq/certs/server_key.pem
+    {{- end }}
+    {{- if .Values.ldap.enabled }}
+    auth_backends.1 = rabbit_auth_backend_ldap
+    auth_backends.2 = internal
+    {{- range $index, $server := .Values.ldap.servers }}
+    auth_ldap.servers.{{ add $index 1 }} = {{ $server }}
+    {{- end }}
+    auth_ldap.port = {{ .Values.ldap.port }}
+    auth_ldap.user_dn_pattern = {{ .Values.ldap.user_dn_pattern  }}
+    {{- if .Values.ldap.tls.enabled }}
+    auth_ldap.use_ssl = true
+    {{- end }}
+    {{- end }}
+    {{- if .Values.metrics.enabled }}
+    ## Prometheus metrics
+    prometheus.tcp.port = 9419
+    {{- end }}
+    {{- if .Values.memoryHighWatermark.enabled }}
+    ## Memory Threshold
+    total_memory_available_override_value = {{ include "rabbitmq.toBytes" .Values.resources.limits.memory }}
+    vm_memory_high_watermark.{{ .Values.memoryHighWatermark.type }} = {{ .Values.memoryHighWatermark.value }}
+    {{- end }}
+    management.ssl.port       = 15671
+    management.ssl.cacertfile = /opt/bitnami/rabbitmq/certs/ca_certificate.pem
+    management.ssl.certfile   = /opt/bitnami/rabbitmq/certs/server_certificate.pem
+    management.ssl.keyfile    = /opt/bitnami/rabbitmq/certs/server_key.pem
+
+  resources:
+    limits:
+      memory: 512Mi
+      cpu: 500m
+    requests:
+      memory: 256Mi
+      cpu: 250m
+
+  persistence:
+    enabled: false
+
+  service:
+    extraPorts:
+      - name: manager-ssl
+        port: 15671
+        targetPort: 15671
+
+  metrics:
+    serviceMonitor:
+      honorLabels: true
+
+  volumePermissions:
+    enabled: true
+    image:
+      repository: bitnami/minideb
+      tag: buster
+      pullPolicy: Always
+      resources:
+        limits:
+          cpu: 100m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 128Mi
+
+# -- Parameters group for bitnami/postgresql helm chart.
+# Details: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/README.md
+# @default -- object
+postgresql:
+  deploy: true
+
+  fullnameOverride: *postgresqlFullName
+
+  image:
+    tag: 11.10.0-debian-10-r9
+
+  postgresqlUsername: *postgresqlUsername
+  postgresqlPassword: *postgresqlPassword
+
+  resources:
+    requests:
+      memory: 256Mi
+      cpu: 250m
+    limits:
+      memory: 512Mi
+      cpu: 500m
+
+  tls:
+    enabled: true
+    certificatesSecret: *tlsSecretName
+    certFilename: "tls.crt"
+    certKeyFilename: "tls.key"
+
+  persistence:
+    enabled: false
```

Comment:
THe same as in https://github.com/cloudify-cosmo/cloudify-helm/pull/56/files#r1016203044

Reply:
Let's keep this one on true

Reply:
persistence enabled for both PostgreSQL and rabbitmq